src/Features/LanguageServer/Protocol/Handler/SignatureHelp/SignatureHelpHandler.cs

```diff
@@ -42,7 +42,7 @@ public override async Task<LSP.SignatureHelp> HandleRequestAsync(LSP.TextDocumen
             var document = context.Document;
             if (document == null)
             {
-                return new LSP.SignatureHelp();
```

Comment:
the nullable annotations need to be updated for the method signature + implementing signature, also in the entry point here - https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs#L426